guides/hhvm/01-installation/06-release-schedule.md

```diff
@@ -11,11 +11,12 @@ Here's a table of recent LTS releases as well as *expected* future LTS releases:
 
 | Version |    \*Release Date       |    \*End of Suppport       |
```

Comment:
```suggestion
| Version |    \*Release Date       |    \*End of Support       |
```

Reply:
wow, nice catch.

Reply:
End of `ppp` support. End of support: today.

Reply:
Ending support very very softly. 🎼